strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -186,7 +324,7 @@ def prepare_gaussian_state(self, r, V, modes):
             )
 
         # convert xp-ordering to symmetric ordering
-        means = vstack([r[:N], r[N:]]).reshape(-1, order="F")
+        means = np.vstack([r[:N], r[N:]]).reshape(-1, order="F")
         C = changebasis(N)
         cov = C @ V @ C.T
```

Comment:
this is place where we could use your new mode and quadrature ordering functions, but maybe we will end up removing/rewriting the whole function.

Reply:
Agreed - we'll probably renovate this function as it's just ported from LocalEngine